projects/coordinator/src/test/java/org/batfish/coordinator/WorkMgrTest.java

```diff
@@ -87,4 +88,41 @@ public void listSortedQuestionNames() throws IOException {
       assertThat(questions.size(), is(3));
       assertThat(questions.toString(), equalTo("[access, initinfo, nodes]"));
    }
+
+   // Tests for initContainer method
```

Comment:
drop this comment -- such comments are unnecessary as the test name should make it clear what it's testing.